shared/build.gradle.kts

```diff
@@ -33,7 +33,7 @@ kotlin {
 
     val xcf = XCFramework()
     listOf(
-//        iosX64(),
+        iosX64(),
```

Comment:
Did you mean to uncomment this?

Reply:
Yeah, was cleaning up the share gradle file and since we need this long term I put it back.